sdk/core/azure-core/src/main/java/com/azure/core/spatial/GeometryBoundingBox.java

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.spatial;
+
+/**
+ * Represents a geometric bounding box.
+ */
+public final class GeometryBoundingBox {
+    private final double west;
+    private final double south;
+    private final double east;
+    private final double north;
+
+    private final Double minAltitude;
+    private final Double maxAltitude;
+
+    /**
+     * Constructs a bounding box.
+     *
+     * @param west West longitudinal boundary.
+     * @param south South latitudinal boundary.
+     * @param east East longitudinal boundary.
+     * @param north North latitudinal boundary.
+     */
+    public GeometryBoundingBox(double west, double south, double east, double north) {
+        this(west, south, east, north, null, null);
+    }
+
+    /**
+     * Constructs a bounding box.
+     *
+     * @param west West longitudinal boundary.
+     * @param south South latitudinal boundary.
+     * @param east East longitudinal boundary.
+     * @param north North latitudinal boundary.
+     * @param minAltitude Minimum altitude boundary.
+     * @param maxAltitude Maximum altitude boundary.
+     */
+    public GeometryBoundingBox(double west, double south, double east, double north, Double minAltitude,
+        Double maxAltitude) {
```

Comment:
Should there be validation on these values?

Reply:
Could be, west/east are longitudes which should be between +/- 180, south/north are latitudes which should be between +/- 90.

Reply:
I wouldn't do any extra validation here. These values can be in meters too. We should keep these object as much of DTOs as possible and let services to the validation.

Reply:
This raises the question on whether we should validate additional constructors as well. For example a polygon type the first and last point in its points need to be the same coordinate, crossing the anti-meridian requires changing type (line to multi-line). The other option is keeping validation localized to simple scenarios such as this.

Reply:
I avoided any validation all together, send whatever customer put into the object and exposed whatever service received. We don't implement any operations on geometries on the client side so we don't care about data being valid in any way.